libminifi/include/core/repository/VolatileRepositoryData.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <vector>
+#include <atomic>
+#include <string>
+#include <memory>
+
+#include "AtomicRepoEntries.h"
+#include "properties/Configure.h"
+
+namespace org::apache::nifi::minifi::core::repository {
+
+struct VolatileRepositoryData {
+  static const char *volatile_repo_max_count;
+  static const char *volatile_repo_max_bytes;
```

Comment:
Please make these top-level const

Reply:
Updated in fb134177002b780f93f09a1c43ebef605cc691e0